quickstart/101-vm-auto-shutdown/readme.md

```diff
@@ -0,0 +1,42 @@
+# Terraform Azure VM Automation Example
```

Comment:
No need to state "Terraform" or "example" as this is the Terraform sample repo. "Azure VM Automation" iis more in keeping with the PG's pattern.

Reply:
Resolved in commit: https://github.com/Azure/terraform/pull/255/commits/51abcf64bfc1b3480c18885ac0ba87fd998a2bd4